docs/source/technical_overview.rst

```diff
@@ -350,27 +350,40 @@ This approach has several advantages over an inheritance-based model:
-between low- and high-dimensional |Models| in pyMOR. The only
+The reduction process in pyMOR is handled by so called :mod:`~pymor.reductors`,
+where each reductor takes arbitrary |Models| of associated type, and additional
+data (e.g. the reduced basis) to create reduced |Models|. For instance, the
+:class:`~pymor.reductors.basic.StationaryRBReductor` and
+:class:`~pymor.reductors.basic.InstationaryRBReductor` reductors can reduce any
+:class:`~pymor.models.basic.StationaryModel` and :class:`~pymor.models.basic.InstationaryModel`,
+while the :class:`~pymor.reductors.bt.GenericBTReductor` reduces
+:class:`~pymor.models.iosys.LTIModel` (just to name a few).
+
+If proper offline/online decomposition is achieved by the reductor, the reduced
+|Model| will not store any high-dimensional data. Note that there is no inherent
+distinction between low- and high-dimensional |Models| in pyMOR. The only
 difference lies in the different types of operators, the |Model|
 contains.
 
 This observation is particularly apparent in the case of the classical
 reduced basis method: the operators and functionals of a given discrete problem
 are projected onto the reduced basis space whereas the structure of the problem
 (i.e. the type of |Model| containing the operators) stays the same.
-pyMOR reflects this fact by offering with :class:`~pymor.reductors.basic.GenericRBReductor`
-a generic algorithm which can be used to RB-project any model available to pyMOR.
-It should be noted however that this reductor is only able to efficiently
-offline/online-decompose affinely |Parameter|-dependent linear problems.
-Non-linear problems or such with no affine |Parameter| dependence require
+pyMOR reflects this fact by offering with :meth:`~pymor.algorithms.projection.project`
+generic algorithms which can be used to RB-project any model available to pyMOR.
+
+In addition to the projection of the |Model|, reductors may also assemble efficient
+offline-online decomposed a posterior error estimates (available via the
+`estimate_error` method of the resulting reduced order |Model|), if more information
+about the underlying problem yielding the full order |Model| is available
+(a popular example is the `CoerciveRBReductor` for |Models| representing stationary
+coercive problems). 
+
+It should be noted that a successfull reduction of more involved |Models| (say those
+arising from non-linear problems or such with no affine |Parameter| dependence) require
 additional techniques such as :mod:`empirical interpolation <pymor.algorithms.ei>`.
 
 If you want to further dive into the inner workings of pyMOR, we
-recommend to study the source code of :class:`~pymor.reductors.basic.GenericRBReductor`
+recommend to study the source code of :class:`~pymor.reductors.basic.ProjectionBasedReductor`
 and to step through calls of it's `reduce` method with a Python debugger, such as
 `ipdb <https://pypi.python.org/pypi/ipdb>`_.
```

Comment:
Is this still a useful advice to give here? I feel we could also defer to the tutorials at this point instead ...

Reply:
Agreed.

Reply:
I'm not sure if `ProjectionBasedReduction` is the right entrypoint as it has not any projection code in it. Maybe point to `StationaryRBReductor.project_operators` instead?